diktat-analysis.yml

```diff
@@ -448,7 +448,7 @@
   enabled: true
 # Checks that outer lambda has explicit parameter name
 - name: PARAMETER_NAME_IN_OUTER_LAMBDA
-  enabled: false
+  enabled: true
 # Checks that property in constructor doesn't contain comment
 - name: KDOC_NO_CONSTRUCTOR_PROPERTY
   enabled: true
```

Comment:
Change all `diktat-analysis.yml` files?

Reply:
nope, will update the rest too

Reply:
just to clarify:
- `diktat-analysis.yml` in root -- is our config file which is used for checking diktat
- `diktat-rules/src/main/resources/diktat-analysis.yml` and `diktat-rules/src/main/resources/diktat-analysis-huawei.yml` -- is config files which are provided in `diktat-rules` as resources as based configuration for checking our consumers